plugin/adsk/scripts/mayaUsd_USDRootFileRelative.py

```diff
@@ -3,18 +3,50 @@
 from mayaUSDRegisterStrings import getMayaUsdString
 from mayaUsdMayaReferenceUtils import pushOptionsUITemplate
 
-class usdRootFileRelative(object):
-    fileNameEditField = None
+class usdFileRelative(object):
+    '''
+    Helper class to create the UI for load/save dialog boxes that need to make the
+    selected file name optionally relative.
 
-    kMakePathRelativeCheckBox = 'MakePathRelative'
+    The caller must tell each function to what the file should be made relative.
```

Comment:
```suggestion
    The caller must tell each function to what the file should be made relative to.
```

Reply:
I'll do the fix in the same commit as the rest to avoid having dozens of commits.